### DISCUSSION TITLE
The color changes during the period that has not yet arrived.

### DISCUSSION THREAD

**Discussion by HuynhDat007** [vkurko/calendar#353] [Q&A]
![image](https://github.com/user-attachments/assets/cde45531-6e93-4347-8050-2641cde1b21d)
The time that has passed will be displayed in one color, while the time that has not yet passed will be displayed in a different color.
Like in the image above.
In mode "Month"
![image](https://github.com/user-attachments/assets/0d1c74bc-5297-4445-b417-28c97b69c6e9)

**Comment by vkurko** [comment]
Unfortunately there is no such setting. You can use [background](https://github.com/vkurko/calendar?tab=readme-ov-file#parsing-event-from-a-plain-object) events on the desired days to change the background color.
